source/backend/discovery/src/lib/sdkResources/index.js

```diff
@@ -94,13 +94,15 @@ async function getAllSdkResources(accountsMap, awsClient, resources) {
     logger.error(`There were ${firstErrors.length} errors when adding first order SDK resources.`);
     logger.debug('Errors: ', {firstErrors});
 
-    firstResults.flat().forEach(resource => resourcesCopy.push(resource));
+    firstResults.flat().forEach(resource => { if(resource != undefined) { resourcesCopy.push(resource) } });
```

Comment:
`!= null` will catch both `null` and `undefined`

Reply:
Actually, we can remove both the null checks if we do this:

```js
const {results: firstResults, errors: firstErrors} = await PromisePool
        .withConcurrency(15)
        .for(resourcesCopy.filter(({resourceType}) => firstOrderResourceTypes.has(resourceType)))
        .process(async resource => {
            const handler = firstOrderHandlers[resource.resourceType];
            return handler(resource);
        })
        .then(({results, errors)) => {
           return {
             errors,
             results: R.reject(R.isNil, results)
           }
         });
```

Reply:
See above: we can remove this check by return an empty array in the DynamoDb handler when `configuration.latestStreamArn` does not exist.